configure

```diff
@@ -773,7 +773,13 @@ def glob_to_var(dir_base, dir_sub):
     (path, dirs, files) = ent
     for file in files:
       if file.endswith('.cpp') or file.endswith('.c') or file.endswith('.h'):
-        list.append('%s/%s' % (dir_sub, file))
+        srcfile = '%s/%s' % (dir_sub, file)
+        if patch_dir:
```

Comment:
I would use `os.path.join` to create path strings so that  we don't have to worry about different operating systems.